Makefile

```diff
@@ -34,6 +34,9 @@ deploy-mainnet: check-api-key deploy
 deploy-rinkeby: export ETH_RPC_URL = $(call network,rinkeby)
 deploy-rinkeby: check-api-key deploy
 
+# verify on Etherscan
+verify:; ETH_RPC_URL=$(call network,$(network_name)) scripts/verify.sh $(contract_address)
```

Comment:
opted into parsing the network name to the command instead of creating a separate one for each network. Would bloat the Makefile IMO.